project/PACKAGERS.md

```diff
@@ -233,6 +233,26 @@ following:
 This will create "./bundles/$VERSION/dynbinary-client/docker-$VERSION", which for
 client-only builds is the important file to grab and install as appropriate.
 
+### Cross Compilation
+
+Limited cross compilation is supported due to requiring cgo for critical
+functionality (such as seccomp support).
+
+To cross compile run `make cross`. You can specify the platforms to target by
+setting the `DOCKER_CROSSPLATFORMS` environment variable to a list of platforms
+in the format `<GOOS>/<GOARCH>`. Specify multiple platforms by using a space
+in between each desired platform.
+
+For setting arm variants, you can specify the `GOARM` value like this:
+
+```
+make DOCKER_CROSSPLATFORMS=linux/arm=7 cross
```

Comment:
was thinking if `armv7` would work (split on `v`), or would that be too much magic?

Reply:
Really it's the same magic, just a different separator.
`v` may be better because of the path in the bundle dir that gets created.

Reply:
that, or use the same parsing as https://github.com/containerd/containerd/blob/1ac546b3c4a3331a9997427052d1cb9888a2f3ef/platforms/database.go#L83

Reply:
Updated as you originally suggested. I like it.